src/html/fetch-external.js

```diff
@@ -13,15 +13,17 @@ const crypto = require('crypto');
 const { setdefault } = require('ferrum');
 const { MountConfig } = require('@adobe/helix-shared');
 
-async function fetchExternal(context, { logger, request, downloader }) {
+async function fetchExternal(context, {
+  logger, request, downloader, secrets,
+}) {
   setdefault(context, 'content', {});
   // check for fstab
   const fstabTask = downloader.getTaskById('fstab');
   if (!fstabTask) {
     logger.info('unable to fetch from external. no fstab task scheduled.');
     return;
   }
-
+  const timeout = secrets.HTTP_TIMEOUT_EXTERNAL || 20000;
```

Comment:
You don't really need the default here and I'd avoid the duplication.

Reply:
ok